lib/pronto/config_file.rb

```diff
@@ -35,8 +36,10 @@ class ConfigFile
       'format' => DEFAULT_MESSAGE_FORMAT
     }.freeze
 
-    def initialize(path = '.pronto.yml')
-      @path = path
+    def initialize
+      @path = ENV.fetch('PRONTO_CONFIG', DEFAULT_FILE_PATH)
+      return if @path == DEFAULT_FILE_PATH || File.exist?(@path)
```

Comment:
@jeroenj I understand it will be set with a value but we should still ensure that the value (the default `.pronto.yml`) is present.  In the code you have we don't bother checking for `.pronto.yml` we just assume it's there.  In the event that isn't, we won't catch it and instead an error will be raised elsewhere.